ckanext/cfpb_extrafields/templates/package/snippets/resource_form.html

```diff
@@ -6,15 +6,7 @@
 {% resource 'cfpb_extrafields/new_resource_redirect.js' %}
 {% if not form_action  %}
 
-<span id='new_resource_redirect'>
-<a href="javascript:;" class="btn" type="button" 
-   data-module="new_resource_redirect"
-   data-module-html="new_resource_redirect.html"
-   data-module-pkgid="{{data.id}}"
-   data-module-pkgname="{{pkg_name}}"
```

Comment:
Ack! Good call. I'll update this PR. I was wondering why it was no longer working locally...